package.json

```diff
@@ -147,5 +148,25 @@
         "outputDirectory": "coverage",
         "outputName": "jest-sonar-report.xml",
         "relativePaths": true
+    },
+    "browserify": {
+        "transform": [
+            "browserify-swap",
+            [
+                "babelify",
+                {
+                    "sourceMaps": "inline",
+                    "presets": [
+                        "@babel/preset-env",
+                        "@babel/preset-typescript"
+                    ]
+                }
```

Comment:
moving this here wasn't a requirement, but the commandline was getting increasingly unwieldy.